### DISCUSSION TITLE
Scrobble API >> Pruple Icon

### DISCUSSION THREAD

**Discussion by frugglehost** [trakt/trakt-api#346] [Q&A] [answered]
Hello,
I'm currently making script to mark shows as they are watched via the Scrobble API.
https://api.trakt.tv/scrobble/start
https://api.trakt.tv/scrobble/stop

On the return from the start API I get a 201 success and similar to the stop code but the ID is blank. I assume this is because the show has just been stopped and nothing is recorded.

On the return from the stop API I get a 201 success and the following telling me that the shows has been added.
`{
    "id":  82******41,
    "action":  "scrobble",
    "progress":  96.0,
    "sharing":  {
                    "facebook":  false,
                    "twitter":  false,
                    "tumblr":  false
                },
    "episode":  {
                    "season":  1,
                    "number":  4,
                    "title":  "The Crossing",
                    "ids":  {
                                "trakt":  5784994,
                                "tvdb":  8827498,
                                "imdb":  "tt15294864",
                                "tmdb":  3408767,
                                "tvrage":  null
                            }
                },
    "show":  {
                 "title":  "1883",
                 "year":  2021,
                 "ids":  {
                             "trakt":  173337,
                             "slug":  "1883",
                             "tvdb":  396390,
                             "imdb":  "tt13991232",
                             "tmdb":  118357,
                             "tvrage":  null
                         }
             }
}`

When I check via the watched API I see the show and time stamp but on the home page Trakt does not show a purple check box next to the episode.
https://api.trakt.tv/sync/watched/shows
`{
    "plays": 4,
    "last_watched_at": "2022-05-06T06:45:27.000Z",
    "last_updated_at": "2022-05-06T06:45:27.000Z",
    "reset_at": null,
    "show": {
        "title": "1883",
        "year": 2021,
        "ids": {
            "trakt": 173337,
            "slug": "1883",
            "tvdb": 396390,
            "imdb": "tt13991232",
            "tmdb": 118357,
            "tvrage": null
        }
    },
    "seasons": [{
        "number": 1,
        "episodes": [{
            "number": 1,
            "plays": 1,
            "last_watched_at": "2022-05-06T05:37:47.000Z"
        }, {
            "number": 2,
            "plays": 1,
            "last_watched_at": "2022-05-06T06:21:23.000Z"
        }, {
            "number": 3,
            "plays": 1,
            "last_watched_at": "2022-05-06T06:40:01.000Z"
        }, {
            "number": 4,
            "plays": 1,
            "last_watched_at": "2022-05-06T06:45:27.000Z"
        }]
    }]
}`

Is there a different API that should be used to add the show as watched once completed?

**Comment by rectifyer** [comment, marked as answer]
The website isn't immediately updated. It checks every 5 minutes to see if new data needs to be synced to the browser.

**Reply by frugglehost** [reply]
Thank you I was impatient on my end.
